selfdrive/ui/qt/spinner.cc

```diff
@@ -115,7 +106,9 @@ void Spinner::update(int n) {
 
 int main(int argc, char *argv[]) {
   setQtSurfaceFormat();
-
+  if (Hardware::EON()) {
```

Comment:
let's make a function like initApp in util for this stuff. it should do all the common stuff like:
* turn on display
* set brightness
* set format
* set gl attributes